index.html

```diff
@@ -124,6 +137,10 @@ <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
     </div>
   </section>
   <section class="about">
+    <div class="images_about">
+    <img src="/img/imgdes/img_about1.png" alt="background_about1" class="img_about1">
+    <img src="/img/imgdes/img_about2.png" alt="background_about2" class="img_about2">
+    </div>
```

Comment:
Seems like the url is not recognized by the browser. Can you please fix the 404 issue for these images to make your app error-free? 😄 

Your Design
<img width="1440" alt="Screenshot 2023-01-21 at 9 24 16 AM" src="https://user-images.githubusercontent.com/24832846/213844631-ed7e2b68-4405-4722-8503-924b55cbc4b7.png">

Figma Design

<img width="1428" alt="Screenshot 2023-01-21 at 9 24 50 AM" src="https://user-images.githubusercontent.com/24832846/213844663-00beb0c5-c9b7-4c4d-b2e0-a821fea2aa82.png">